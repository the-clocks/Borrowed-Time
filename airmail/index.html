<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <title>Air Mail Clock - BorrowedTime</title>
  <link rel="preconnect" href="https://fonts.googleapis.com">
  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
  <link href="https://fonts.googleapis.com/css2?family=Libre+Baskerville&family=Nanum+Gothic+Coding:wght@400;700&family=Oxanium:wght@200..800&family=Roboto+Slab:wght@100..900&family=Syne+Mono&display=swap" rel="stylesheet">
  <style>
    @font-face {
      font-family: 'air';
      src: url('air.ttf') format('truetype');
    }

    body {
      background-color: #15abcc;
      color: rgb(84, 82, 176);
      display: flex;
      justify-content: center;
      align-items: center;
      height: 100vh;
      margin: 0;
      overflow: hidden;
    }

    .clock {
      font-family: 'air';
      display: flex;
    }

    .digit-box {
      width: 2rem;
      display: flex;
      justify-content: center;
      align-items: center;
      font-size: 8rem;
      box-sizing: border-box;
    }

    .colon {
      width: 1rem;
      height: 7rem;
      display: flex;
      justify-content: center;
      align-items: center;
      font-size: 7rem;
    }

    .stamp2, .stamp, .stamp3 {
      position: absolute;
      z-index: 2;
      transform-origin: center center;
    }


    .stamp3 {
      top: 0.2rem;
      right: 0.01rem;
      width: 8rem;
      height: 4rem;
    }

    .stamp2 {
      top: 3.5rem;
      right: 1rem;
      width: 5rem;
      height: 4rem;
    }

    .stamp {
      top: 0.5rem;
      right: 7.3rem;
      width: 7rem;
      height: 5rem;
    }

    .bgimage {
      position: absolute;
      top: 0;
      left: 0;
      width: 100%;
      height: 100%;
      z-index: -1;
      transform-origin: center center;
      opacity: 0.75;
    }

    .title-container {
      color: #a1d1ef;
      text-shadow: #eeeff0 1px 0px;
      position: absolute;
      top: 5px;
      left: 50%;
      transform: translateX(-50%);
      width: 98%;
      display: flex;
      z-index: 6;
    }

    .chltitle {
      font-family: "Roboto Slab", serif;
      font-size: 2.8vh;
      position: absolute;
      top: 1px;
      right: 1px;
      letter-spacing: 0.1vh;
    }

    .bttitle {
      position: relative;
      left: 0;
      font-family: "Oxanium", serif;
      font-size: 2.8vh;
      font-style: italic;
      z-index: 15;
      letter-spacing: -0.1vh;
    }

    a {
      color: inherit;
      text-decoration: none;
    }

    a:hover {
      color: #e8ecec;
      background-color: rgb(21, 0, 255);
      text-decoration: none;
    }

    .date-container {
      color: #09598b;
      position: absolute;
      bottom: 5px;
      left: 50%;
      transform: translateX(-50%);
      width: 98%;
      display: flex;
      z-index: 6;
    }

    .clockname {
      font-family: "Oxanium", serif;
      position: fixed;
      left: 50%;
      transform: translateX(-50%);
      font-size: 4vh;
      line-height: 4vh;
    }

    .dateleft, .dateright {
      font-size: 3vh;
      font-family: "Nanum Gothic Coding", monospace;
    }

    .dateleft {
      position: relative;
      left: 0;
    }

    .dateright {
      position: absolute;
      right: 0;
    }

    /* Jostle Animation */
    @keyframes bounceJostle {
      0%, 10%, 20%, 90%, 100% {
        transform: translate(0, 0) rotate(0);
      }
      5%   { transform: translate(1px, -1.5px) rotate(0.3deg); }
      12%  { transform: translate(-1.2px, 1.2px) rotate(-0.25deg); }
      18%  { transform: translate(1.5px, 0.8px) rotate(0.2deg); }
      25%  { transform: translate(-1px, -1px) rotate(-0.2deg); }
      40%  { transform: translate(0.5px, 0.3px) rotate(0.1deg); }
      55%  { transform: translate(0, 0) rotate(0); }
      70%  { transform: translate(-1.2px, 1.2px) rotate(0.25deg); }
      75%  { transform: translate(1.5px, -1.5px) rotate(-0.3deg); }
      82%  { transform: translate(-0.7px, 1px) rotate(0.15deg); }
    }

    .jostle {
      animation: bounceJostle 10s infinite ease-in-out;
    }

    .stamp.jostle {
      animation-delay: 0.4s;
    }

    .stamp2.jostle {
      animation-delay: 0.8s;
    }

    .bgimage.jostle {
      animation-delay: 0.2s;
    }
  </style>
</head>
<body>
  <div class="title-container">
    <div class="chltitle">Cubist Heart Laboratories</div>
    <div class="bttitle">BorrowedTime</div>
  </div>

  <div class="date-container">
    <a href="../elephant/" class="dates dateleft">05/31/25</a>
    <a href="../index.html" class="clockname">Air Mail</a>
    <a href="../dashboard" class="dates dateright">06/02/25</a>
  </div>
  <img src="./stamp3.png" class="stamp3 jostle">

  <img src="./stamp2.png" class="stamp2 jostle">
  <img src="./stamp.png" class="stamp jostle">

 
  <img src="./frame.jpg" class="bgimage jostle">

  <div class="clock" id="clock">
    <!-- Digits will be inserted by JavaScript -->
  </div>

  <script>
    const clock = document.getElementById('clock');

    // Create digit and colon boxes
    const format = ['digit', 'digit', 'colon', 'digit', 'digit', 'colon', 'digit', 'digit'];
    const boxes = format.map(type => {
      const div = document.createElement('div');
      div.className = type === 'colon' ? 'colon jostle' : 'digit-box jostle';
      clock.appendChild(div);
      return div;
    });

    function updateClock() {
      const now = new Date();
      const timeStr = now.toLocaleTimeString('en-US', { hour12: false });
      const timeDigits = timeStr.replace(/:/g, '').split('');
      let i = 0;
      for (let j = 0; j < boxes.length; j++) {
        boxes[j].textContent = format[j] === 'colon' ? ':' : timeDigits[i++];
      }
    }

    updateClock();
    setInterval(updateClock, 1000);
  </script>
</body>
</html>
