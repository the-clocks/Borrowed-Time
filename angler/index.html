<!doctype html>
<html lang="en">

<head>
    <title>Anglerfish Clock - BorrowedTime</title>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0/dist/js/bootstrap.bundle.min.js"></script>
    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet"
        integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
    <link rel="preconnect" href="https://fonts.googleapis.com">
    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
    <link href="https://fonts.googleapis.com/css2?family=Oxanium:wght@200..800" rel="stylesheet">
    <link href="https://fonts.googleapis.com/css2?family=Barriecito&display=swap" rel="stylesheet">

    <style>
        * {
            margin: 0;
            padding: 0;
            box-sizing: border-box;
        }

        html {
            height: 100%;
            width: 100%;
            display: flex;
            justify-content: center;
            align-items: center;
            overflow: hidden;
        }

        body {
            background-color: #074060;
        }

        .content {
            font-family: "Barriecito", Impact, Haettenschweiler, 'Arial Narrow Bold', sans-serif;
            background: linear-gradient(90deg, #369b91, #0e8c68, #711579);
            background-repeat: no-repeat;
            background-size: 75%;
            animation: animate 3s linear infinite;
            background-clip: text;
            opacity: 0.5;
            -webkit-text-fill-color: rgba(255, 255, 255, 0);
            white-space: nowrap;
            z-index: 9;
            position: relative;
        }

        @keyframes animate {
            0% {
                background-position: -500%;
            }

            50% {
                background-position: 500%;
            }

            100% {
                background-position: 100%;
            }
        }

        .overlay {
            position: fixed;
            top: 0;
            left: 0;
            width: 100vw;
            height: 100vh;
            opacity: 0.9;
            background: url(./images/anglerfish-gif_anglerfish_idle_swim.webp) no-repeat center center;
            background-size: 100% 100%;
            z-index: 1;
        }

        .overlay2 {
            position: fixed;
            top: 0;
            left: 0;
            width: 100vw;
            height: 100vh;
            opacity: 0.4;
            background: url(./images/Deep-Sea-Anglerfish-Fuse.webp) no-repeat center center fixed;
            background-size: 100% 100%;
            z-index: 2;
        }

        .overlay3 {
            position: fixed;
            top: 0;
            left: 0;
            width: 100vw;
            height: 100vh;
            opacity: 0.4;
            background: url(./images/qsxwwd.webp) no-repeat center center;
            background-size: 33% 33%;
            background-repeat: repeat;
            z-index: 4;
        }

        .overlay4 {
            position: fixed;
            top: 0;
            left: 0;
            width: 100vw;
            height: 100vh;
            opacity: 0.3;
            background: url(./images/spin.gif) no-repeat center center fixed;
            -webkit-transform: scaleX(-1);
            transform: scaleX(-1);
            background-repeat: no-repeat;
            background-size: 100% 100%;
            z-index: 4;
        }


        a {
            color: inherit;
            text-decoration: none;
        }

        a:hover {
            color: #e8ecec;
            background-color: rgb(21, 0, 255);
            text-decoration: none;
        }

        .date-container {
            color: #7a7bcf;
            position: absolute;
            bottom: 5px;
            left: 50%;
            transform: translateX(-50%);
            width: 98%;
            display: flex;
            z-index: 6;
        }

        .clockname {
            font-family: "Oxanium", serif;
            position: fixed;
            left: 50%;
            transform: translateX(-50%);
            font-family: "Oxanium", serif;
            font-size: 4vh;
            line-height: 4vh;
        }


        .dateleft {
            font-size: 3vh;
            font-family: "Nanum Gothic Coding", monospace;
            position: relative;
            left: 0;
        }

        .dateright {
            font-size: 3vh;
            font-family: "Nanum Gothic Coding", monospace;
            position: absolute;
            right: 0;
        }

        .title-container {
            color: #0e0c0d;
            text-shadow: #9dd9ed 1px 0px;
            position: absolute;
            top: 5px;
            left: 50%;
            transform: translateX(-50%);
            width: 98%;
            display: flex;
            /* justify-content: space-between; */
            z-index: 6;
        }

        .chltitle {
            font-family: "Roboto Slab", serif;
            font-size: 2.8vh;
            position: absolute;
            top: 1px;
            right: 1px;
            letter-spacing: 0.1vh;
        }

        .bttitle {
            position: relative;
            left: 0;
            font-family: "Oxanium", serif;
            font-size: 2.8vh;
            font-style: italic;
            z-index: 15;
            letter-spacing: -0.1vh;
        }

        @media (min-aspect-ratio: 1/1) {
            .content {
                font-size: calc(100vw / 5);
                transform: scaleY(calc(100vh / (10 * 1em)));
            }
        }

        @media (max-aspect-ratio: 1/1) {
            .content {
                font-size: calc(100vh / 5);
                transform: scaleX(calc(100vw / (10 * 1em)));
            }
        }



        #fadein {
            position: fixed;
            top: 0;
            left: 0;
            width: 100%;
            height: 100%;
            background: black;
            z-index: 9999;
            opacity: 1;
            transition: opacity 1s ease;
            /* slow fade */
        }

        #fadein.fade-out {
            opacity: 0;
            pointer-events: none;
        }
    </style>

</head>

<body>
    <div id="fadein"></div>
    <div class="title-container">
        <div class="chltitle">Cubist Heart Laboratories</div>
        <div class="bttitle">BorrowedTime</div>
    </div>

    <div class="date-container">
        <a href="../room/" class="dates dateleft">04/09/25</a>
        <a href="../index.html" class="clockname">Anglerfish Clock</a>
        <a href="../index.html" class="dates dateright">04/11/25</a>
    </div>

    <div class="content" id="clock"></div>

    <script>
        function updateClock() {
            const now = new Date();
            let hours = now.getHours();
            let minutes = now.getMinutes();

            hours = hours % 24;
            minutes = minutes < 10 ? '0' + minutes : minutes;

            const time = '' + hours + ' ' + minutes + '';
            document.getElementById('clock').textContent = time;
            setTimeout(updateClock, 1000);
        }

        updateClock();
    </script>
    <script>
        window.addEventListener('load', () => {
            setTimeout(() => {
                document.getElementById('fadein').classList.add('fade-out');
            }, 500); // wait 0.5 seconds before starting the slow fade
        });
    </script>
    <div class="overlay"></div>
    <div class="overlay2"></div>
    <div class="overlay3"></div>
    <div class="overlay4"></div>

</body>

</html>