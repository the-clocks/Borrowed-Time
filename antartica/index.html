<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <title>Antartica Clock - BorrowedTime</title>
    <link rel="preconnect" href="https://fonts.googleapis.com">
    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
    <link
        href="https://fonts.googleapis.com/css2?family=Libre+Baskerville&family=Nanum+Gothic+Coding:wght@400;700&family=Oxanium:wght@200..800&family=Roboto+Slab:wght@100..900&family=Syne+Mono&display=swap"
        rel="stylesheet">

    <style>
        body {
            display: flex;
            justify-content: center;
            align-items: center;
            height: 100vh;
            margin: 0;
        }

        .clock {
            position: relative;
            width: 50vh;
            height: 30vh;
            border-radius: 50%;
        }

        .tick {
            position: absolute;
            width: 0.1vw;
            height: 37.5vh;
            background-color: #b6eef6;
            top: 1.2vh;
            left: 50%;
            transform-origin: 50% 14.5vh;
        }

        .tick.major {
            height: 58vh;
            width: 0.1vw;
        }

        .hand {
            position: absolute;
            bottom: 50%;
            left: 50%;
            transform-origin: bottom;
            background-color: #b3edf2;
        }

        .hour-hand {
            width: 3.7vw;
            height: 17vh;

        }

        .minute-hand {
            width: 2vw;
            height: 33vh;
        }

        .second-hand {
            width: 0.3vw;
            height: 100vh;
            background-color: rgb(72, 219, 242);
        }

        .center {
            position: absolute;
            width: 1.5vh;
            height: 1.5vh;
            background-color: #333;
            border-radius: 50%;
            top: 50%;
            left: 50%;
            transform: translate(-50%, -50%);
        }

        .title-container {
            color: #427290;
            text-shadow: #42afef 1px 0px;
            position: absolute;
            top: 5px;
            left: 50%;
            transform: translateX(-50%);
            width: 98%;
            display: flex;
            /* justify-content: space-between; */
            z-index: 6;
        }

        .chltitle {
            font-family: "Roboto Slab", serif;
            font-size: 2.8vh;
            position: absolute;
            top: 1px;
            right: 1px;
            letter-spacing: 0.1vh;
        }

        .bttitle {
            position: relative;
            left: 0;
            font-family: "Oxanium", serif;
            font-size: 2.8vh;
            font-style: italic;
            z-index: 15;
            letter-spacing: -0.1vh;
        }

        a {
            color: inherit;
            text-decoration: none;
        }

        a:hover {
            color: #e8ecec;
            background-color: rgb(21, 0, 255);
            text-decoration: none;
        }

        .date-container {
            color: #89b9d6;
            position: absolute;
            bottom: 5px;
            left: 50%;
            transform: translateX(-50%);
            width: 98%;
            display: flex;
            /* justify-content: space-between; */
            z-index: 6;
        }

        .clockname {
            font-family: "Oxanium", serif;
            position: fixed;
            left: 50%;
            transform: translateX(-50%);
            font-family: "Oxanium", serif;
            font-size: 4vh;
            line-height: 4vh;
        }

        .dateleft {
            font-size: 3vh;
            font-family: "Nanum Gothic Coding", monospace;
            position: relative;
            left: 0;
        }

        .dateright {
            font-size: 3vh;
            font-family: "Nanum Gothic Coding", monospace;
            position: absolute;
            right: 0;
        }

        @keyframes slow-rotate {
            0% {
                transform: rotate(0deg) scale(1.5);
            }

            100% {
                transform: rotate(360deg) scale(1.5);
            }
        }

        .bgimage {
            position: absolute;
            top: 0;
            left: 0;
            width: 100%;
            height: 100%;
            object-fit: cover;
            filter: contrast(0.4);
            z-index: -1;
            animation: slow-rotate 900s linear infinite;
            transform-origin: center center;
        }
    </style>
</head>

<body>
    <div class="title-container">
        <div class="chltitle">Cubist Heart Laboratories</div>
        <div class="bttitle">BorrowedTime</div>
    </div>
    <img src="./images/Antarctica Satellite Image.jpg" class="bgimage">
    <div class="date-container">
        <a href="../stripe/" class="dates dateleft">04/17/25</a>
        <a href="../index.html" class="clockname">Antartica Clock</a>
        <a href="../index.html" class="dates dateright">04/19/25</a>
    </div>

    <div class="clock">
        <div class="center"></div>
    </div>
    <script>
        // Create tick marks
        const clock = document.querySelector('.clock');
        for (let i = 0; i < 60; i++) {
            const tick = document.createElement('div');
            tick.className = 'tick';
            if (i % 5 === 0) tick.className += ' major';
            tick.style.transform = `rotate(${i * 6}deg)`;
            clock.appendChild(tick);
        }

        // Create hands
        const hourHand = document.createElement('div');
        hourHand.className = 'hand hour-hand';
        const minuteHand = document.createElement('div');
        minuteHand.className = 'hand minute-hand';
        const secondHand = document.createElement('div');
        secondHand.className = 'hand second-hand';
        clock.appendChild(hourHand);
        clock.appendChild(minuteHand);
        clock.appendChild(secondHand);

        // Update clock
        function updateClock() {
            const now = new Date();
            const hours = now.getHours() % 12;
            const minutes = now.getMinutes();
            const seconds = now.getSeconds();
            const milliseconds = now.getMilliseconds();

            const hourAngle = (hours * 30) + (minutes / 2);
            const minuteAngle = (minutes * 6) + (seconds / 10);
            const secondBaseAngle = seconds * 6;
            const progress = milliseconds / 1000;
            const secondAngle = secondBaseAngle + (progress < 0.3 ? progress * 12 : 0); // Overshoot effect

            hourHand.style.transform = `translateX(-50%) rotate(${hourAngle}deg)`;
            minuteHand.style.transform = `translateX(-50%) rotate(${minuteAngle}deg)`;
            secondHand.style.transform = `translateX(-50%) rotate(${secondAngle}deg)`;

            requestAnimationFrame(updateClock);
        }

        updateClock();
    </script>
</body>

</html>