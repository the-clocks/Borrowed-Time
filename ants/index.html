<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>align-items Clock - BorrowedTime</title>
    <link rel="preconnect" href="https://fonts.googleapis.com">
    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
    <link
        href="https://fonts.googleapis.com/css2?family=Lora:ital,wght@0,400..700;1,400..700&family=Nanum+Gothic+Coding&family=Oxanium:wght@200..800&family=Roboto+Slab:wght@100..900&display=swap"
        rel="stylesheet">
    <style>
        @font-face {
            font-family: 'Ant';
            src: url('Ant.ttf') format('truetype');
        }

        body {
            display: flex;
            justify-content: center;
            align-items: center;
            height: 100vh;
            margin: 0;
        }

        .clock {
            width: 45vh;
            height: 45vh;
            max-width: 400px;
            max-height: 400px;
            min-width: 200px;
            min-height: 200px;
            border-radius: 50%;
            position: relative;
            z-index: 6;
        }

        .number {
            font-family: 'Ant', sans-serif;
            position: absolute;
            width: 100%;
            height: 100%;
            text-align: center;
            font-size: 21vh;
            ;
        }

        .number span {
            display: inline-block;
            transform: translateY(-17.5vw);
            width: 8vh;
            height: 12vw;
            line-height: 1vw;
            text-align: center;
        }

        .hand {
            position: absolute;
            bottom: 50%;
            left: 50%;
            transform-origin: bottom;
            z-index: 10;
            /* Below numbers */
        }

        .hour-hand img {
            width: 2vw;
            /* Scales with clock */
            /* color:blanchedalmond; */
            height: 29vw;
            max-width: 8px;
            max-height: 80px;
            min-width: 4px;
            min-height: 40px;
        }

        .minute-hand img {
            width: 1.5vw;
            height: 25vw;
            max-width: 6px;
            max-height: 100px;
            min-width: 3px;
            min-height: 50px;
        }

        .second-hand img {
            width: 1vw;
            height: 30vw;
            max-width: 4px;
            max-height: 120px;
            min-width: 2px;
            min-height: 60px;
        }

        .center {
            width: 3vw;
            /* Scales with clock */
            height: 3vw;
            max-width: 12px;
            max-height: 12px;
            min-width: 6px;
            min-height: 6px;
            background-color: rgb(237, 9, 9);
            border-radius: 50%;
            position: absolute;
            top: 50%;
            left: 50%;
            transform: translate(-50%, -50%);
            z-index: 20;
            /* Below numbers, above hands */
        }


        .title-container {
            color: #fbf5f8;
            text-shadow: #f90a0a 1px 0px;
            position: absolute;
            top: 5px;
            left: 50%;
            transform: translateX(-50%);
            width: 98%;
            display: flex;
            z-index: 6;
        }

        .chltitle {
            font-family: "Roboto Slab", serif;
            font-size: 2.1vh;
            position: absolute;
            top: 1px;
            right: 1px;
            letter-spacing: 0.1vh;
        }

        .bttitle {
            position: relative;
            left: 0;
            font-family: "Oxanium", serif;
            font-size: 2.7vh;
            font-style: italic;
            letter-spacing: -0.1vh;
        }

        a {
            color: inherit;
            text-decoration: none;
        }

        a:hover {
            color: #e8ecec;
            background-color: rgb(21, 0, 255);
            text-decoration: none;
        }

        .date-container {
            color: #000000;
            position: absolute;
            bottom: 5px;
            left: 50%;
            transform: translateX(-50%);
            width: 98%;
            display: flex;
            z-index: 60;
        }

        .clockname {
            font-family: "Oxanium", serif;
            position: fixed;
            left: 50%;
            transform: translateX(-50%);
            font-size: 3.5vh;
            line-height: 4vh;
        }

        .dateleft {
            font-size: 3vh;
            font-family: "Nanum Gothic Coding", monospace;
            position: relative;
            left: 0;
        }

        .dateright {
            font-size: 3vh;
            font-family: "Nanum Gothic Coding", monospace;
            position: absolute;
            right: 0;
        }

        .bgimage {
            position: absolute;
            top: 0;
            left: 0;
            width: 100%;
            height: 100%;
            object-fit: cover;
            z-index: 1;

        }

        .bgimage2 {
            position: fixed;
            top: 0;
            left: 0;
            width: 100%;
            height: 100%;
            background-image: url('./ants.gif');
            /* fixed escape */
            background-repeat: repeat-x;
            /* tile horizontally only */
            background-size: auto 100%;
            /* keep original width, stretch height */
            z-index: 2;
            pointer-events: none;
        }
    </style>
</head>

<body>
    <div class="title-container">
        <div class="chltitle">Cubist Heart Laboratories</div>
        <div class="bttitle">BorrowedTime</div>
    </div>

    <div class="date-container">
        <a href="../blueprint/" class="dates dateleft">05/18/25</a>
        <a href="../index.html" class="clockname">Ants</a>
        <a href="../index.html" class="dates dateright">05/20/25</a>
    </div>
    <div class="bgimage2"></div>
    <div class="clock" id="clock"></div>
    <img src="./ants-clipart-animation.gif" class="bgimage">

    <script>
        function createClock() {
            const clock = document.getElementById('clock');

            for (let i = 1; i <= 12; i++) {
                const angle = i * 30; // 30째 per number
                const number = document.createElement('div');
                number.className = 'number';

                const span = document.createElement('span');
                span.textContent = i;
                span.style.backgroundColor = i % 2 === 0 ? 'transparent' : 'transparent'; // Even = red, Odd = black
                span.style.color = 'black'; // All numbers yellow

                number.appendChild(span);
                number.style.transform = `rotate(${angle}deg)`;
                clock.appendChild(number);
            }


            // Create hands with placeholder images
            const hands = [
                { class: 'hour-hand', img: 'https://placehold.co/8x80/000000/000000/png' },
                { class: 'minute-hand', img: 'https://placehold.co/6x100/000000/000000/png' },
                { class: 'second-hand', img: 'https://placehold.co/4x120/FF0000/FF0000/png' }
            ];
            hands.forEach(hand => {
                const div = document.createElement('div');
                div.className = `hand ${hand.class}`;
                const img = document.createElement('img');
                img.src = hand.img;
                img.style.display = 'block';
                div.appendChild(img);
                clock.appendChild(div);
            });

            // Create center dot
            const center = document.createElement('div');
            center.className = 'center';
            clock.appendChild(center);
        }


        function updateClock() {
            const now = new Date();
            const hours = now.getHours() % 12;
            const minutes = now.getMinutes();
            const seconds = now.getSeconds();

            const hourDeg = (hours + minutes / 60) * 30; // 30째 per hour
            const minuteDeg = (minutes + seconds / 60) * 6; // 6째 per minute
            const secondDeg = seconds * 6; // 6째 per second

            document.querySelector('.hour-hand').style.transform = `translateX(-50%) rotate(${hourDeg}deg)`;
            document.querySelector('.minute-hand').style.transform = `translateX(-50%) rotate(${minuteDeg}deg)`;
            document.querySelector('.second-hand').style.transform = `translateX(-50%) rotate(${secondDeg}deg)`;
        }

        createClock();
        updateClock();
        setInterval(updateClock, 1000);
    </script>
</body>

</html>