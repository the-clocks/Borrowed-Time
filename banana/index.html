<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
  <title>Banana Clock with Images and Numbers</title>
  <style>
    body {
      margin: 0;
      background: #804e4e;
      display: flex;
      justify-content: center;
      align-items: center;
      height: 100vh;
    }

    .clock {
      position: relative;
      width: 95vw;
      height: 95vw;
      max-width: 90vh;
      max-height: 90vh;
      font-family: sans-serif;
      color: white;
    }

    .clock-face {
      width: 100%;
      height: 100%;
      object-fit: contain;
      position: absolute;
      top: 0;
      left: 0;
    }

    .hand {
      position: absolute;
      width: 50%;
      height: 50%;
      top: 25%;
      left: 25%;
      transform-origin: 50% 100%;
      pointer-events: none;
    }

    .hour-hand { z-index: 3; }
    .minute-hand { z-index: 2; }
    .second-hand { z-index: 1; }

    .number {
      position: absolute;
      width: 10%;
      text-align: center;
      transform: translate(-50%, -50%);
      font-size: 3vw;
      font-weight: bold;
      pointer-events: none;
    }
  </style>
</head>
<body>
  <div class="clock">
    <img src="clock-face.png" alt="Clock Face" class="clock-face">
    <img src="hour-hand.png" alt="Hour Hand" class="hand hour-hand" id="hourHand">
    <img src="minute-hand.png" alt="Minute Hand" class="hand minute-hand" id="minuteHand">
    <img src="second-hand.png" alt="Second Hand" class="hand second-hand" id="secondHand">

    <!-- Clock numbers 1â€“12 -->
    <div class="number" style="top: 10%; left: 50%;">12</div>
    <div class="number" style="top: 19%; left: 73%;">1</div>
    <div class="number" style="top: 36%; left: 87%;">2</div>
    <div class="number" style="top: 50%; left: 92%;">3</div>
    <div class="number" style="top: 64%; left: 87%;">4</div>
    <div class="number" style="top: 81%; left: 73%;">5</div>
    <div class="number" style="top: 90%; left: 50%;">6</div>
    <div class="number" style="top: 81%; left: 27%;">7</div>
    <div class="number" style="top: 64%; left: 13%;">8</div>
    <div class="number" style="top: 50%; left: 8%;">9</div>
    <div class="number" style="top: 36%; left: 13%;">10</div>
    <div class="number" style="top: 19%; left: 27%;">11</div>
  </div>

  <script>
    function updateClock() {
      const now = new Date();
      const seconds = now.getSeconds();
      const minutes = now.getMinutes();
      const hours = now.getHours() % 12;

      const secondDeg = seconds * 6;
      const minuteDeg = minutes * 6 + seconds * 0.1;
      const hourDeg = hours * 30 + minutes * 0.5;

      document.getElementById('secondHand').style.transform = `rotate(${secondDeg}deg)`;
      document.getElementById('minuteHand').style.transform = `rotate(${minuteDeg}deg)`;
      document.getElementById('hourHand').style.transform = `rotate(${hourDeg}deg)`;
    }

    setInterval(updateClock, 1000);
    updateClock();
  </script>
</body>
</html>
