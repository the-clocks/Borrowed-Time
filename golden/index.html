<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>Responsive Digital Clock</title>
    <link rel="preconnect" href="https://fonts.googleapis.com">
    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
    <link
        href="https://fonts.googleapis.com/css2?family=Metamorphous&family=Nanum+Gothic+Coding&family=Oxanium:wght@200..800&family=Roboto+Slab:wght@100..900&display=swap"
        rel="stylesheet">
    <style>
        @font-face {
            font-family: 'Shiny';
            src: url('fonts/Shiny.ttf') format('truetype');
        }

        html,
        body {
            margin: 0;
            padding: 0;
            height: 100vh;
            width: 100vw;
            display: flex;
            justify-content: center;
            align-items: center;
            /* background-color: #5b5858; */
            /* color: rgb(231, 217, 67); */

        }

        .frame {
            position: relative;
            width: 70vw;
            height: 70vh;
            /* border: 0.5vh solid white; */
            display: flex;
            justify-content: center;
            align-items: center;
            z-index: 2;
            opacity: 0.9;
            box-shadow: 0 0 10px rgba(237, 200, 18, 0.5);

        }

        /* .rectangle-frame {
            position: absolute;
            top: -1vh;
            bottom: -1vh;
            left: -1vw;
            right: -1vw;
            border: 4.5vh dashed #db2828;
            pointer-events: none;
            position: fixed;
            top: 0;
            left: 0;
            width: 100vw;
            height: 100vh;
            z-index: 3;
        } */

        .clock {
            font-family: 'Shiny';
            color: #fcfbf8;
            text-shadow: rgb(213, 102, 80) 1px 1px 1px;
            display: flex;
            gap: 5vw;
            font-size: 18vw;
            line-height: 24vw;
            text-align: center;
            z-index: 3;
        }

        .time-unit {
            width: 20vw;
        }

        @media (max-width: 600px) {
            .clock {
                flex-direction: column;
                /* font-size: 10vw; */
            }

            .time-unit {
                width: 100%;
            }
        }

        .bgimage {
            position: fixed;
            top: 0;
            left: 0;
            width: 100vw;
            height: 100vh;
            z-index: 1;
            filter: brightness(120%);
            -webkit-transform: scaleX(-1);
            transform: scaleX(-1);
        }

        a {
            color: inherit;
            text-decoration: none;
        }

        a:hover {
            color: #e8ecec;
            background-color: rgb(21, 0, 255);
            text-decoration: none;
        }

        .date-container {
            color: #e3d7f9;
            position: absolute;
            bottom: 5px;
            left: 50%;
            transform: translateX(-50%);
            width: 98%;
            display: flex;
            z-index: 9;
        }

        .clockname {
            font-family: "Oxanium", serif;
            position: fixed;
            left: 50%;
            transform: translateX(-50%);
            font-family: "Oxanium", serif;
            font-size: 4vh;
            line-height: 4vh;
        }


        .dateleft {
            font-size: 3vh;
            font-family: "Nanum Gothic Coding", monospace;
            position: relative;
            left: 0;
        }

        .dateright {
            font-size: 3vh;
            font-family: "Nanum Gothic Coding", monospace;
            position: absolute;
            right: 0;
        }


        .title-container {
            color: #dbe3e8;
            text-shadow: #e90f46 1px 0px;
            position: absolute;
            top: 5px;
            left: 50%;
            transform: translateX(-50%);
            width: 98%;
            display: flex;
            /* justify-content: space-between; */
            z-index: 9;
        }

        .chltitle {
            font-family: "Roboto Slab", serif;
            font-size: 2.1vh;
            position: absolute;
            top: 1px;
            right: 1px;
            letter-spacing: 0.1vh;
        }

        .bttitle {
            position: relative;
            left: 0;
            font-family: "Oxanium", serif;
            font-size: 2.7vh;
            font-style: italic;
            letter-spacing: -0.1vh;
        }
    </style>
</head>

<body>
    <div class="title-container">
        <div class="chltitle">Cubist Heart Laboratories</div>
        <div class="bttitle">BorrowedTime</div>
    </div>
    <div class="date-container">
        <a href="../radar/" class="dates dateleft">05/07/25</a>
        <a href="../index.html" class="clockname">Golden Hour Clock</a>
        <a href="../whirlpool/" class="dates dateright">05/09/25</a>
    </div>
    <img src="./images/d7e781b32269a8a82b500c1a9dc97733-ezgif.com-optimize.gif" class="bgimage">


    <div class="frame">
        <div class="clock">
            <div class="time-unit" id="hours">00</div>
            <div class="time-unit" id="minutes">00</div>
        </div>
    </div>
    <script>
        function updateClock() {
            const now = new Date();
            const pad = n => String(n).padStart(2, '0');
            document.getElementById('hours').textContent = pad(now.getHours());
            document.getElementById('minutes').textContent = pad(now.getMinutes());
        }

        setInterval(updateClock, 1000);
        updateClock();
    </script>
</body>

</html>